spec/javascript/packages/document-capture-polling/index-spec.js

```diff
@@ -70,7 +71,10 @@ describe('DocumentCapturePolling', () => {
 
     expect(trackEvent).to.have.been.calledOnceWithExactly(
       'IdV: Link sent capture doc polling started',
-      { phone_question_ab_test_bucket: 'bypass_phone_question' },
+      {
+        phone_question_ab_test_bucket: 'bypass_phone_question',
+        phone_with_camera: false,
```

Comment:
I think this will need to be added to the attributes in the events where it's used

https://github.com/18F/identity-idp/blob/9201a1112b04c554c437ec08fe94d163e07821e7/app/services/analytics_events.rb#L2232-L2236

Reply:
I wonder if we should have a "hybrid" path or branch in `spec/fetaures/idv/analytics_spec.rb` to check for a more "real" usage. Between this and https://github.com/18F/identity-idp/pull/9481#discussion_r1376745597, it doesn't seem like our current tests are giving us much assurance that these events are actually getting logged as expected.